src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -702,7 +703,7 @@ SparqlValues Visitor::visit(Parser::InlineDataFullContext* ctx) {
                 "No variables were specified in Values "
                 "clause. This is not supported by QLever.");
   for (auto& var : ctx->var()) {
-    values._variables.push_back(visit(var).name());
+    values._variables.push_back(Variable{visit(var).name()});
```

Comment:
Same here.

Reply:
I think then the whole loop can just be
`values._variables = visitVector(ctx->var());`

Reply:
You are so right!